data/json/vitrification/unvitrified_farm.json

```
@@ -0,0 +1,288 @@
+      ],
+      "palettes": [ "standard_domestic_palette", "standard_domestic_basement_palette" ],
+      "furniture": { "!": "f_table" },
+      "place_nested": [
+        { "chunks": [ "utility_3x3_S" ], "x": 8, "y": 7 },
+        { "chunks": [ "basement_storage_1x1" ], "x": [ 5, 17 ], "y": [ 3, 6 ], "repeat": [ 4, 16 ] }
+      ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "unvitrified_farm_2",
+    "object": {
+      "fill_ter": "t_shingle_flat_roof",
+      "rows": [
+        "                        ",
+        "                        ",
+        "    ---------------     ",
+        "    -.............-     ",
+        "    -.............-     ",
+        "    -.............-     ",
+        "    -.............-     ",
+        "    -.............-     ",
+        "    -.............-     ",
+        "    -.............-     ",
+        "    ---5..........-     ",
+        "       -..........-     ",
+        "       -..........-     ",
+        "       -----......-     ",
+        "           -......-     ",
+        "           --------     ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        "
+      ],
+      "palettes": [ "roof_palette" ],
+      "terrain": { ".": "t_shingle_flat_roof" }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "unvitrified_orchard" ],
+    "weight": 100,
+    "object": {
+      "set": [ { "point": "variable", "id": "unvitrified_orchard_center", "x": 12, "y": 12 } ],
+      "fill_ter": "t_dirt",
+      "rows": [
+        "111111111111111111111111",
+        "!**********************1",
+        "!*    *    **    *    *1",
+        "!* 7  7  7 ** 7  7  7 *1",
+        "!*         **         *1",
+        "!*         **         *1",
+        "!**7     7****7     7**1",
+        "!*         **         *1",
+        "!*         **         *1",
+        "!* 7  7  7 ** 7  7  7 *1",
+        "!*    *    **    *    *1",
+        "@**********************1",
+        "@**********************1",
+        "!*    *    **    *    *1",
+        "!* 7  7  7 ** 7  7  7 *1",
+        "!*         **         *1",
+        "!*         **         *1",
+        "!**7     7****7     7**1",
+        "!*         **         *1",
+        "!*         **         *1",
+        "!* 7  7  7 ** 7  7  7 *1",
+        "!*    *    **    *    *1",
+        "!**********************1",
+        "111111111111111111111111"
+      ],
+      "palettes": [ "orchard_tree_apple" ],
+      "terrain": { "1": "t_quietfarm_border_fence", "!": "t_fence", "@": "t_fencegate_c" }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "compass direction indicates head of bed",
+    "nested_mapgen_id": "bedroom_4x4_child_vit",
```

```suggestion
    "method": "json",
    "nested_mapgen_id": "bedroom_4x4_child_vit",
```
You've not even provided a compass direction c;